zz/zzfilme.sh

```diff
@@ -0,0 +1,65 @@
+# ----------------------------------------------------------------------------
+# Busca informações sobre o filme desejado.
+# Use a opção -t ou --traducao para ver a sinopse traduzida pelo Google
+# Translate.
+#
+# Uso:	zzfilme [-t|--traducao] FILME
+# Ex.:	zzfilme matrix
+#		zzfilme 'matrix revolutions'
+#		zzfilme -t 'lord of the rings'
+#
+# Autor: Vinícius Venâncio Leite <vv.leite (a) gmail com>
+# Desde: 2018-04-16
+# Versão: 1
+# Licença: GPL
+# Requisitos: zzxml, zztrim, zztradutor, zzecho
+# ----------------------------------------------------------------------------
+zzfilme ()
+{
+	zzzz -h filme "$1" && return
+
+	test -n "$1" || { zztool -e uso filme; return 1; }
+	
+	local traducao=0
+	local tab=$(printf '\t')
+	local api='https://www.omdbapi.com/?apikey=2c30e4db&type=movie&r=xml&plot=full&t'
+	
+	while test -n "$1"
+	do
+		case "$1" in
+			-t | --traducao)
+				local traducao=1
+				shift
+			;;
+			*)
+				local filme="$1"
+				shift
+			;;
+		esac
+	done
+	
+	local request=$(zztool dump "https://www.omdbapi.com/?apikey=2c30e4db&type=movie&r=xml&plot=full&t=$filme" | zzxml --tag movie --indent | sed 's/\("\) /\1\n/g ; s/<movie // ; s/\/>// ; s/"//g' | zztrim)
+	
+	if test -z "$request"
```

Comment:
Antes desse ponto, se a sugestão do uso da ``` zzurlencode ``` for usada, aqui seria interessante usar a ``` zzurldecode ``` para redefinir de novo a variável filme